src/components/modal/modal.ts

```diff
@@ -150,8 +150,10 @@ class BXModal extends HostListenerMixin(LitElement) {
   open = false;
 
   render() {
-    const containerClasses = classnames(`${prefix}--modal-container`, {
-      [this.containerClass]: this.containerClass,
+    const containerClass = this.containerClass.split(' ').reduce((acc, item) => ({ ...acc, [item]: true }), {});
```

Comment:
```suggestion
    const containerClass = this.containerClass.split(' ').filter(Boolean).reduce((acc, item) => ({ ...acc, [item]: true }), {});
```

My apologies for not suggesting this earlier 🙇

Reply:
Oh no please, thank you your solution is by far better than what I was doing hehe